README.md

```diff
@@ -103,8 +107,19 @@ for more detail on querying.
 
 ## Optional
```

Comment:
Seen as this section is now all about Kibana I suggest changing the heading to `Kibana` rather than `Optional`.